fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/ExistentialApply2OuterJoinRule.java

```diff
@@ -289,9 +289,7 @@ public List<OptExpression> transform2OuterJoin(OptimizerContext context, OptExpr
 
         // filter(UnCorrelation) agg -> project -> un-correlation filter
         if (null != apply.getPredicate()) {
-            OptExpression filterOptExpression =
-                    OptExpression.create(new LogicalFilterOperator(apply.getPredicate()), input.getInputs().get(1));
```

Comment:
so when to add `input.getInputs().get(1)`?

Reply:
duplicate code in line 299